### PR TITLE
What’s New for 5.0

### PR DESCRIPTION
**Pull request by bhariharan**
@rwhitley and I reviewed it offline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +21,30 @@ This pulls submodule dependencies from github.
...
+
+**OS Version Support**
+- Android Nougat (API 25) is fully supported in Mobile SDK 5.0.
 
 **SmartStore Enhancements**
-- To enhance performance in certain edge cases, SmartStore adds an option for serializing unusually large soup elements in external storage.
+- We have added new APIs that allow hybrid developers to create their own named databases.
+
+**SmartSync Enhancements**
+- We now allow sync down/refresh of data stored in local soups by specifying the fields to sync.
```
Comment (wmathurin) [inline README.md]: I'm not sure about "local" soups since all soups are local.

File: README.md [hunk 2]
```diff
@@ -21,16 +21,30 @@ This pulls submodule dependencies from github.
...
+
+**Hybrid Networking Enhancements**
+- We have replaced forcetk.mobilesdk.js with force.js. Networking in hybrid apps is now handled natively. As a result, session token refresh happens automatically.
```
Comment (wmathurin) [inline README.md]: Should we indicate that: Networking in hybrid apps is now handled natively **through a new plugin (com.salesforce.plugin.network)**.

Comment (wmathurin) [inline README.md]: Should we also indicate that smartsync.js no longer depends on jQuery but instead uses browser promises.

**Comment by bhariharan** [comment]
Done [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1323/commits/e2096f606f2ebb691c03885fdb94c1e5a7d70daf).

**Pull request merged**